src/app/web-app.component.ts

```diff
@@ -113,6 +126,141 @@ export class WebAppComponent implements OnInit {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Create Group
+        'Control.Shift.g': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Create Center
+        'Control.Shift.q': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Frequent Postings
+        'Control.Shift.f': (event: KeyboardEvent): void => {
+          this.router.navigate(['accounting/journal-entries/frequent-postings']);
+          event.preventDefault();
+        },
+
+        // Closure Entries
+        'Control.Shift.e': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Journal Entries
+        'Control.Shift.j': (event: KeyboardEvent): void => {
+          this.router.navigate(['accounting/journal-entries']);
+          event.preventDefault();
+        },
+
+        // Reports
+        'Control.Shift.r': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Accounting
+        'Control.Shift.a': (event: KeyboardEvent): void => {
+          this.router.navigate(['accounting']);
+          event.preventDefault();
+        },
+
+        // Save/Submit Forms
+        'Control.s': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Run Report
+        'Control.r': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Cancel
+        'Control.Shift.x': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Logout
+        'Control.Shift.l': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Search
+        'Control.Shift.h': (event: KeyboardEvent): void => {
+          this.router.navigate(['help']);
+          event.preventDefault();
+        },
+
+        // Pagination: Next
+        'Control.n': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
```

Comment:
Various tables have been implemented in the accounting section. Figure out a way to make this shortcut work there. Make sure that it possible with angular material.